Ammo.xml repath

WrillWasTaken opened the pull request:
this is not a place of honor. there is no glory here. this is a dark, terrible place of our creation.

Commit:
Update medtool_overrides.xml

Commit:
port pomegrenade extract changes

Commit:
increase ether tank capacity

Commit:
update surgical equipment

Commit:
remove medkits

Commit:
adjust styptic powder numbers

ven1883 committed:
Merge pull request #22 from ven1883/medical-update

Update Medical

Commit:
Port Jobs, unbungle texts

Jobs and texts are painfully interlinked. Not implying everything else isn't, but this was the straw that made me get this done.

ven1883 committed:
Merge pull request #23 from ven1883/port-jobs

Port Jobs, unbungle texts

Commit:
bring over descriptions update

ven1883 committed:
Merge pull request #26 from ven1883/texts-update

bring over descriptions update, remove unused texts

Commit:
KILL PAIN SHITCODE

WE ARE SAVED

Commit:
change pain indicatorlimb to head

more convenient

Commit:
OOPS

forgot to set the affliction back to pain after I used internaldamage for testing

Commit:
double all pain

ven1883 committed:
Merge pull request #27 from ven1883/lua-pain

KILL PAIN SHITCODE

Commit:
repathing and reformatting!!!!!!!!!!

immense agony

Commit:
more repathing

does the suffering end or does it merely transform

Commit:
more reformat

why is there an m249????????????

Commit:
curiosity

Commit:
more format

in my hubris, I have sinned.

Commit:
formatte!!!!!!!!!!

Commit:
commit

ven1883 commented on Items/Weapons/Ammo/smg_ammo.xml:
```diff
@@ -0,0 +1,427 @@
+    <Holdable canBeCombined="true" slots="Any,RightHand,LeftHand" handle1="0,0" msg="ItemMsgPickUpSelect" />
+
+  </Item>
+
```
missing separation indicators, use !--###-- comments

WrillWasTaken replied:
done

ven1883 commented on Items/Weapons/Ammo/smg_ammo.xml:
```diff
@@ -0,0 +1,427 @@
+    </Projectile>
+
+  </Item>
+
```
some items do not have names attached to them, do this!!

ven1883 reviewed (changes requested):
also you need to clean out and remove the ammo.xml file itself otherwise I have to do it manually

WrillWasTaken commented:
> also you need to clean out and remove the ammo.xml file itself otherwise I have to do it manually

![image](https://github.com/ven1883/Europa-Corps/assets/86113712/77e3e0c8-ff63-45e9-9875-67b03131dc49)

Commit:
dividers

Commit:
special ammo names

Commit:
rifle ammo naming

Commit:
explosive + shotgun naming

Commit:
the singular missing pistol round name

Commit:
smg naming

Commit:
heavy naming

ven1883 committed:
sketch

fixes some whitespace sketch primarily

yes it was easier to PR this

ven1883 committed:
explosive_ammo.xml patch

please replicate some of this in the other files

ven1883 commented:
i gaveth you a PR to work with

also the fixme file was NOT deleted!!!!!!!!! it is in my PR

ven1883 committed:
Merge pull request #29 from ven1883/axml-patch

AXML patch

ven1883 committed:
Update explosive_ammo.xml

ven1883 committed:
updates heavy_ammo.xml

ven1883 committed:
Update explosive_ammo.xml

I sense a great disturbance in the force.

ven1883 commented:
before i go insane wrill asked me to finish this

ven1883 committed:
Update pistol_ammo.xml

fuck

ven1883 committed:
woohoo yeah wooo yeahhh!!!

death

ven1883 committed:
smg hell

so back

Commit:
remove eha bullshit

we literally do not need a single one of these

Commit:
remove eha bullshit

we literally do not need a single one of these

ven1883 committed:
ufkcing shurgun

ven1883 committed:
special weapons fix!!

speshul

Commit:
this code is misery

what did humanity do, to deserve demons as vile as the language they call "xml"?

Commit:
this code is misery

what did humanity do, to deserve demons as vile as the language they call "xml"?

Commit:
reformat med overrides

Commit:
reformat med overrides

Commit:
minor adjustment

Commit:
minor adjustment

Commit:
Update medtools.xml

let's just make sure I'm not stupid enough to do this in the future

Commit:
Update medtools.xml

let's just make sure I'm not stupid enough to do this in the future

Commit:
yet more reformatting

Commit:
yet more reformatting

Commit:
finish up the medical folder

AAAAAAAAAAAAAAAAAAAAAAAAAAAAA
(i still need to repath it)

Commit:
finish up the medical folder

AAAAAAAAAAAAAAAAAAAAAAAAAAAAA
(i still need to repath it)

Commit:
we don't need this

Commit:
we don't need this

Commit:
Update materials.xml

Commit:
Update materials.xml

Commit:
throw some stuff around

Commit:
throw some stuff around

Commit:
Revert "throw some stuff around"

This reverts commit f9cee93fa36df4b00ba0886dd9ce46b3d9150f0f.

Commit:
Revert "Update materials.xml"

This reverts commit 34003b550981d0da71af2201471121b0e6132af5.

Commit:
Revert "we don't need this"

This reverts commit d22405400ad1080c2f64f590d551508ca47f14d5.

Commit:
Revert "finish up the medical folder"

This reverts commit 4919d7ca7cc6638057a948dc92d1b15c65e87eff.

Commit:
Revert "yet more reformatting"

This reverts commit 90d98ba21d2bb8de1177a486a734ae0dd968bc71.

Commit:
Revert "Update medtools.xml"

This reverts commit 3a518f1aaf990ff050f6aebea46ac948ab4c5ee9.

Commit:
Revert "minor adjustment"

This reverts commit 3e1aec23f5ea552f0fd331ece7a69ade1aa6b634.

Commit:
Revert "reformat med overrides"

This reverts commit 8d05653c5a37a48db5ad05c7bc4ddc2ebb4aadd9.

Commit:
Revert "this code is misery"

This reverts commit 40fa37ae399bebe8af81859f43e51f7a5ca5139b.

Commit:
Revert "remove eha bullshit"

This reverts commit 08a93b91337a59fb6acba4d76afb11053518c311.

ven1883 committed:
Merge pull request #30 from ven1883/reformatting-hell

Reformatting hell

Commit:
add armor sprites

Commit:
move everything into their respective files

Commit:
Delete armor.xml

Commit:
reformat coalition.xml

Commit:
Delete special.xml

Commit:
reformat separatists.xml

Commit:
move scp_combathelmet to coalition.xml

Commit:
reformat universal.xml

Commit:
reformat other.xml

Commit:
add armor files to filelist, repath armor textures

Commit:
repath separatists.xml

Commit:
Update separatists.xml

Commit:
Update universal.xml

Commit:
Update other.xml

Commit:
readd accidentally deleted sprites

Commit:
Update universal.xml

Commit:
more repathing

Commit:
updates

Commit:
hell

Commit:
more stupid dumb shit

Commit:
EVEN MORE

Commit:
add animations

Commit:
yet more updates

Commit:
repath textures

ven1883 committed:
Merge pull request #31 from ven1883/armor-repath-reformat

Armor repath & reformat and reloc... yeah, basically just everything that needs to be done with armor.

Commit:
Update filelist.xml

Commit:
Merge branch 'main' into medical-update-repath

Commit:
buh

Commit:
HATE

ven1883 committed:
Merge pull request #32 from ven1883/medical-update-repath

Medical updates and repathing (and formatting!), so yeah, basically everyth-

Commit:
mild formatting cleanup

Commit:
remove autofill + crafting from melee

Commit:
Update alien.xml

Commit:
Update alien.xml

Commit:
Update explosives.xml

Commit:
Update heavy.xml

Commit:
Update pistols.xml

Commit:
change 556 ammo + extended to use variants

Commit:
Update rifle_ammo.xml

Commit:
smite the free market

the wrath of RegEx

Commit:
wrath of regex be upon ye

why doesn't java 8 have \X??? trying to find a substitute for this was 90% of the development time for this one regex string

Commit:
fully reformat rifles.xml

XML? you mean XMHELL?

Also, I opted to take the easy route and do lots of en masse changes with regex and changing all occurences of certain stuff.

Commit:
Merge branch 'main' into ammo.xml_repath

Commit:
Update filelist.xml

Commit:
stuff

Commit:
reformat + repathing

Commit:
Update special.xml

Commit:
repath heaven

I LOVE REGEX RAHHHHHHHHHHH
(?:(texture|file)="%ModDir.*?%/[^iIaAeEhJjLlTtPp])

Commit:
repath heaven

Commit:
add targets to statuseffects

Commit:
some progress on sound files

ven1883 committed:
proletariat

 A spectre is haunting Europe — the spectre of communism. All the powers of old Europe have entered into a holy alliance to exorcise this spectre: Pope and Tsar, Metternich and Guizot, French Radicals and German police-spies.

Where is the party in opposition that has not been decried as communistic by its opponents in power? Where is the opposition that has not hurled back the branding reproach of communism, against the more advanced opposition parties, as well as against its reactionary adversaries?

Two things result from this fact:

I. Communism is already acknowledged by all European powers to be itself a power.

II. It is high time that Communists should openly, in the face of the whole world, publish their views, their aims, their tendencies, and meet this nursery tale of the Spectre of Communism with a manifesto of the party itself.

Chapter I. Bourgeois and Proletarians

ven1883 committed:
turret ammo

the only error

Commit:
Merge branch 'main' into items-repath

Commit:
Merge branch 'ammo.xml_repath'

ven1883 reviewed (commented):
Review system is goofy
